packages/api-server/api_server/repositories/tasks.py

```diff
@@ -24,6 +23,20 @@
 from api_server.query import add_pagination
 
 
+class TaskFavoriteIn(BaseModel):
+    id: str
+    name: str
+    unix_millis_earliest_start_time: datetime
+    priority: Dict
+    category: str
+    description: Dict
+    user: str
+
+
+class TaskFavoriteOut(TaskFavoriteIn):
+    unix_millis_earliest_start_time: int
```

Comment:
Are these being models being used somewhere? I can't seem to find any references to them